src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -16,15 +16,19 @@ public class CommandResult {
     /** Help information should be shown to the user. */
     private final boolean showHelp;
 
+    /** Client information should be shown to the user. */
+    private final boolean showClient;
+
     /** The application should exit. */
     private final boolean exit;
```

Comment:
Should these boolean variables be named as `isExit` , `isShowHelp`, `isShowClient` instead?